app/src/main/java/org/p2p/wallet/home/ui/main/HomeFragment.kt

```diff
@@ -218,13 +219,9 @@ class HomeFragment :
 
     override fun showTokensForBuy(tokens: List<Token>, newBuyEnabled: Boolean) {
         if (newBuyEnabled) {
-            NewSelectTokenBottomSheet.show(
-                fm = childFragmentManager,
-                title = getString(R.string.buy_select_token_title),
-                tokens = tokens,
-                requestKey = KEY_REQUEST_TOKEN,
-                resultKey = KEY_RESULT_TOKEN
-            )
+            tokens.find { it.tokenSymbol == Constants.USDC_SYMBOL }?.let { token ->
```

Comment:
are we opening NewBuyFragment only for USDC?

Reply:
USDC is default token to buy